src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs

```diff
@@ -674,6 +769,26 @@ private IConfidentialClientApplication BuildConfidentialClientApplication(Merged
             }
         }
 
+        /// <summary>
+        /// Creates a managed identity client application.
+        /// </summary>
+        /// <param name="managedIdentityId">Indicates if system-assigned or user-assigned managed identity is used.</param>
+        /// <param name="enablePiiLogging">Indicates if logging that may contain personally identifiable information should be enabled.</param>
+        /// <returns>A managed identity application.</returns>
+        private IManagedIdentityApplication BuildManagedIdentityApplication(ManagedIdentityId managedIdentityId, bool enablePiiLogging)
+        {
+            return ManagedIdentityApplicationBuilder
+                .Create(managedIdentityId)
+                .WithHttpClientFactory(_httpClientFactory)
```

Comment:
As discussed with @jmprieur , we'd like to bypass the HTTP pipeline customization for MSI. The reason for this that in MSIv2 (which @gladjohn is working on), there will be an MTLS connection happening, which doesn't play well with this extensibility point (we can go into details if needed)

The HTTP calls will be made to local service endpoints anyway, we don't think there are scenarios where apps would want to customize this. 

We can always add this on.

Reply:
If we take this later, is the comment still blocking or what are you looking for as a resolution?

Reply:
or just remove `WithHttpClient`...seems to be the suggestion in another comment?